cmd/makenew/new_checker.go.tmpl

```diff
@@ -0,0 +1,53 @@
+package lint
+
+import (
+	"go/ast"
+	"go/token"
+
+    "github.com/go-toolsmith/astcopy"
+	"github.com/go-toolsmith/astequal"
+	"github.com/go-toolsmith/astfmt"
+	"github.com/go-toolsmith/astp"
```

Comment:
Seems like there are issues with import lines alignment.
It's not a big deal, since this is a template, but it looks ugly.